src/Tribe/iCal.php

```diff
@@ -275,8 +275,8 @@ public function generate_ical_feed( $post = null, $echo = true ) {
 			$item[] = 'CREATED:' . $tzoned->created;
 			$item[] = 'LAST-MODIFIED:' . $tzoned->modified;
 			$item[] = 'UID:' . $event_post->ID . '-' . $time->start . '-' . $time->end . '@' . parse_url( home_url( '/' ), PHP_URL_HOST );
-			$item[] = 'SUMMARY:' . str_replace( array( ',', "\n", "\r", "\t" ), array( '\,', '\n', '', '\t' ), html_entity_decode( strip_tags( $event_post->post_title ), ENT_QUOTES ) );
-			$item[] = 'DESCRIPTION:' . str_replace( array( ',', "\n", "\r", "\t" ), array( '\,', '\n', '', '\t' ), html_entity_decode( strip_tags( $event_post->post_content ), ENT_QUOTES ) );
+			$item[] = 'SUMMARY:' . str_replace( array( ',', "\n", "\r", "\t" ), array( '\,', '\n', '', '	' ), html_entity_decode( strip_tags( $event_post->post_title ), ENT_QUOTES ) );
```

Comment:
I only did that since I saw the "\t" text within the macOS Calendar app during real-world use with how it was set up previously, and my update to change `\t` to `	` seemed to fix that problem (for some reason).

The update to leave the tab as-is (remove from the str_replace array) & simplify the code as a result has been made.